content/en/dashboards/sharing.md

```diff
@@ -19,7 +19,39 @@ further_reading:
 
 ## Overview
 
-Shared graphs and screenboards allow you to display metric, trace, and log visualizations outside of Datadog.
+Shared dashboards and graphs allow you to display metric, trace, and log visualizations outside of Datadog.
+
+## Dashboards
+
+**Share an entire dashboard publicly by generating a URL**:
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
+3. Choose configuration options under "Time & Variable Setings" for "Allow changing timeframe" and/or template variables visible tags.
+4. Copy the URL and click **Done**.
+
+**Share with individual emails**: 
+1. On the dashboard's page, click the settings cog in the upper right.
+2. Choose the **Generate public URL** option.
```

Comment:
```suggestion
2. Select **Generate public URL**.
```
Side comment, it seems really weird usability-wise to have to select an option with the word "public" in it to send a private URL, no?